docs/adr/ADR-009-cw-fees.md

```diff
@@ -0,0 +1,64 @@
+---
+
+# ADR-009: Introduction of CWFees Module
+
+Date: [Insert Date]
+
+## Status
+
+[Insert Status: Proposed, Accepted, etc.]
```

Comment:
Status can be Accepted / Implemented I presume...

Reply:
updated!